org.nextcloud.Nextcloud.appdata.xml

```diff
@@ -1,22 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
-  <id>org.nextcloud.Nextcloud.desktop</id>
+  <id>org.nextcloud.Nextcloud</id>
   <name>Nextcloud</name>
   <project_license>GPL-2.0+</project_license>
   <summary>Nextcloud sync client</summary>
-  <metadata_license>AGPL</metadata_license>
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
@TingPing @juliushaertl Are you fine with relicensing this file under a more permissive license? 
For more info about why it should be relicensed see: https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-metadata_license

Reply:
Fine by me :+1: